docs/technical_reference_guide/README.md

```diff
@@ -44,6 +44,7 @@ mkdir _build
 
 ```
 latexmk -pdf -latexoption=-file-line-error -latexoption=-interaction=nonstopmode -output-directory=_build -halt-on-error ResStockTechnicalReferenceGuide.tex 
+```
 
 6. All of the pdf and log files from the compile will be located in docs/technical_reference_guide/_build (Note: this won't show up as modified files in git because _build is in .gitignore)
-```
+
```

Comment:
7. If the pdf does not compile successfully, look for fatal errors in the .log file, apply fixes, and recompile as needed by repeating step 5. To stop the workflow and exit out of the container, type `exit` in the CLI or hit `Ctrl + D` on your keyword.

Reply:
addressed: [997cc80](https://github.com/NREL/resstock/pull/1349/commits/997cc808591bda43d35d0448645363985df1a1d9)